src/strings.cpp

```diff
@@ -272,34 +273,21 @@ char *GetStringWithArgs(char *buffr, StringID string, StringParameters *args, co
 
 	if (index >= _langpack.langtab_num[tab]) {
 		if (game_script) {
-			return GetStringWithArgs(buffr, STR_UNDEFINED, args, last);
+			return GetStringWithArgs(builder, STR_UNDEFINED, args);
 		}
 		FatalError("String 0x{:X} is invalid. You are probably using an old version of the .lng file.\n", string);
 	}
 
-	StringBuilder builder(buffr, last);
 	FormatString(builder, GetStringPtr(string), args, case_index);
-	return builder.GetEnd();
 }
 
 char *GetString(char *buffr, StringID string, const char *last)
 {
 	_global_string_params.ClearTypeInformation();
 	_global_string_params.offset = 0;
-	return GetStringWithArgs(buffr, string, &_global_string_params, last);
-}
-
-/**
- * Get a parsed string with most special stringcodes replaced by the string parameters.
- * @param builder     The builder of the string.
- * @param string      The ID of the string to parse.
- * @param args        Arguments for the string.
- * @param case_index  The "case index". This will only be set when FormatString wants to print the string in a different case.
- * @param game_script The string is coming directly from a game script.
- */
-void GetStringWithArgs(StringBuilder &builder, StringID string, StringParameters *args, uint case_index = 0, bool game_script = false)
-{
-	builder.AddViaStreCallback([&](auto buff, auto last) { return GetStringWithArgs(buff, string, args, last, case_index, game_script); });
+	StringBuilder builder(buffr, last);
```

Comment:
buffr .. lol .. I always wonder what made someone spell it like that. "No, the extra `e` makes the sentence WAY too long".

No action required. Just funny.

Reply:
I hope that function is gone by the end of the month.